Statistics/Distributions/LogNormal.cs

```diff
@@ -68,18 +74,19 @@ private void addRules()
         #region Functions
         #region IDistribution
         public override double PDF(double x){
-            Normal sn = new Normal();
-            return sn.PDF(Math.Log(x));
+            Normal standardNormal = new Normal();
```

Comment:
we should use the Normal.StandardNormal(p) instead

Reply:
all I see in the normal class is `public static double StandardNormalInverseCDF(double p)` I do not see a Normal.StandardNormal(p)